open_academy/i18n/zh_CN.po

```diff
@@ -0,0 +1,250 @@
+# Translation of Odoo Server.
```

Comment:
Why do you use this localization? (the file name zh_CN.po, indicates the localization).

When we don't know what localization the user has for their language (if they are from Spain, Mexico, Colombia), we use the es.po file. This will allow the user to use any localization for the Spanish language, it will have our translation.

Please delete this file.